.github/workflows/test.yml

```
@@ -40,8 +40,7 @@ jobs:
     env:
       # The "default" npm is the one that ships with a given version of node.
       # For more: https://nodejs.org/en/download/releases/
-      IS_MRT_NODE: ${{ matrix.node == 16 && matrix.npm == 8 }}
-      IS_DEFAULT_NPM: ${{ matrix.node == 16 && matrix.npm == 8 || matrix.node == 18 && matrix.npm == 9 }}
+      IS_DEFAULT_NPM: ${{ (matrix.node == 16 && matrix.npm == 8) || (matrix.node == 18 && matrix.npm == 9) }}
       IS_LATEST_NPM: ${{ matrix.node == 16 && matrix.npm == 8 }}
```

This `IS_LATEST_NPM` is no longer correct. In the [original intention](https://github.com/SalesforceCommerceCloud/pwa-kit/commit/b743269099c5dce5a77587653ad93d9639ab428b#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R43), it was set to npm 8 because the other variable (`IS_DEFAULT_NPM`) has npm 8 as the latest version. 

But now that we're supporting Node 18 and npm 9 (#1112) , should it be this instead?

```suggestion
      IS_LATEST_NPM: ${{ matrix.node == 18 && matrix.npm == 9 }}
```

---

@vmarta lighthouse CI (`lhci`) doesn't work with `node18` + `npm9` at this time:

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/v3/.github/workflows/test.yml#L67-L71

---

(this was verified via numerous failed test runs, the `lhci` binary times out and won't execute 100% of the time)

---

The lighthouse tests aren't part of this job, so that's not really a concern here. But, like you called out in your [other comment](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1126/files#r1163582246), we should probably be a bit more clear/intentional about what we use the env vars for.

For example, `IS_DEFAULT_NPM` was originally for conditionally including the step "install a different npm". But it's also used for "run this step only once per version of node". It might be more clear if we had an env var with the same condition, but named `IS_ONCE_PER_NODE_VERSION`. Or maybe just more detailed comments. Similarly, `IS_MRT_NODE` and `IS_LATEST_NPM` are both slightly different use cases of "once per CI run". I dunno if the best solution is better names or better comments, but we should have _something_ to make the intentions explicit / not require knowledge of anything outside this file.

---

Sounds like there are some ambiguities around this and based on @bfeister's comment, we don't want node 18 yet. Plus this is not a part of this PR, i'm just going to resolve the comment if ya'll are okay.